src/data/calculate.ts

```diff
@@ -276,7 +313,8 @@ export const calculateActivityRisk = (data: CalculatorData): number | null => {
     if (multiplier > MAX_ACTIVITY_RISK) {
       multiplier = MAX_ACTIVITY_RISK
     }
-    return multiplier
+
+    return multiplier * vaccineMultiplier
```

Comment:
God. This is just such a satisfying line to read. It's like.... go humanity! We did it! We have vaccines!

Reply:
👩‍⚕️💉😎